src/components/FeaturedSlider/SliderButton.jsx

```diff
@@ -0,0 +1,10 @@
+/* eslint-disable react/prop-types */
```

Comment:
This could be a todo for further deliverables, prop types are a bonus option, but are a good practice with React stand alone with JS